### PR TITLE
Convert lua strings to files

### PR DESCRIPTION
**Pull request by theis999**
### Just for show

Lua code is moved to separate files. Those files are set to output on build.

### File settings
The files will be moved from the TAS-helper to the TAS mod on script generation. Script files that up to date is skipped, except the goal file and the step file.
The goal file changes based on which goal you choose, but the copy_from file remains unchanged which is why it needs to be updated.

### Lua setup
The goals are split into separate files. 
control.lua now require a goal.lua file
control1 and control2 is merged together

### CODE REVIEW

**Comment by MortenTobiasNielsen** [comment]
Is this up to date with the master branch?

**Comment by theis999** [comment]
It was as of 2 hours ago

File: info.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+	"name": "test_1.0",
+	"version": "0.0.2",
```
Comment (MortenTobiasNielsen) [inline info.json:3]: This should be version 0.0.5

**Review by MortenTobiasNielsen — CHANGES_REQUESTED** [review]

**Comment by MortenTobiasNielsen** [comment]
control_info.h should be cleaned or deleted.

File: info.json (hunk 1 above)
Comment (theis999) [reply info.json:3]: Correct. Looking at it, info.json can't be made this way. Since we need to change the mod name.

Comment (MortenTobiasNielsen) [reply info.json:3]: Ohh yeah you are right. Info might live best in c++. :)

Comment (theis999) [reply info.json]: Agreed, i reverted to the old way.

**Comment by theis999** [comment]
I have reverted to the old info.json and cleaned the control_info.h

**Pull request merged**